assets/js/contact.js

```diff
@@ -12,6 +12,7 @@ function bindCaptchaValidation() {
     if (!captcha || !captcha.value) {
       event.preventDefault();
       error.textContent = error.dataset.error;
+      error.className = error.className.replace(/(\s*)display-none(\s*)/, '$1$2');
```

Comment:
WDYT of using `classList.remove()`? Looks like it has decent support? https://caniuse.com/?search=classlist

```suggestion
      error.classList.remove('display-none');
```

Reply:
Ah! I mentioned it in my original comment, but I totally forgot there is partial support in Internet Explorer, at least for what we need. I'll update.